sympy/combinatorics/pc_groups.py

```diff
@@ -0,0 +1,58 @@
+from sympy.core import Basic
+from sympy import sieve
+from sympy.combinatorics.perm_groups import PermutationGroup
+
+class PcGroup(Basic):
```

Comment:
I think `PolycyclicGroup` sounds better. Why `PcGroup`? Can you give a short explanation? Just asking out of curiosity.